module/Company/src/Company/Form/CollectionBaseFieldsetAwareForm.php

```diff
@@ -14,19 +14,11 @@ class CollectionBaseFieldsetAwareForm extends Form
     // This implementation fixes it
     public function bind($object, $flags = FormInterface::VALUES_NORMALIZED)
     {
-        if (!in_array($flags, array(FormInterface::VALUES_NORMALIZED, FormInterface::VALUES_RAW))) {
-            throw new InvalidArgumentException(
-                sprintf(
-                    '%s expects the $flags argument to be one of "%s" or "%s"; received "%s"',
-                    __METHOD__,
-                    'Laminas\Form\FormInterface::VALUES_NORMALIZED',
-                    'Laminas\Form\FormInterface::VALUES_RAW',
-                    $flags
-                )
-            );
+        if (!in_array($flags, [FormInterface::VALUES_NORMALIZED, FormInterface::VALUES_RAW])) {
+            throw new InvalidArgumentException(sprintf('%s expects the $flags argument to be one of "%s" or "%s"; received "%s"', __METHOD__, 'Laminas\Form\FormInterface::VALUES_NORMALIZED', 'Laminas\Form\FormInterface::VALUES_RAW', $flags));
         }
```

Comment:
In certain cases these longer methods (most often exceptions) are compressed into a single line. This is something we should try to prevent.

Reply:
I agree. I'll see what I can do.